src/web/client/extension.ts

```diff
@@ -268,12 +269,95 @@ export function processWillSaveDocument(context: vscode.ExtensionContext) {
     );
 }
 
-export function processWillStartCollaboartion() {
+export function processWillStartCollaboartion(context: vscode.ExtensionContext) {
     if (isCoPresenceEnabled()) {
-        // TODO: Add copresence logic
+        processOnTabChange(context);
+        createWebWorkerInstance(context);
     }
 }
 
+let worker: Worker;
```

Comment:
lets add this in webextension context - we dont keep global variables like this.

Reply:
I tried this earlier but worker was not working correctly! can we pick this up in any subsequent PR?
when we have UI for complete visibility and testing

Reply:
Done: https://github.com/microsoft/powerplatform-vscode/pull/745/commits/b2568e6f21e2e06594b429e8f91af27f0bd4d778